installation/GRCh38_related_files/calc_gc_content.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import argparse
+import re
+
+assert sys.version_info.major >= 3, "Python 3 is required for string operations" 
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-i", "--input", help="Input sequence - fasta file format", required=True)
+parser.add_argument("-o", "--output", help="Output file name", default='gc_content.txt')
+parser.add_argument("-t", "--threshold", help="Threshold", default=0.0)
+parser.add_argument("--stepsize", help="Threshold (regions with gc lower than threshold are discarded)", default=10000)
+parser.add_argument("-v", "--verbose", action = 'store_true', help="Add verbosity")
+args = parser.parse_args()
+
+threshold = float(args.threshold)
+stepsize = int(args.stepsize)
+
+output_data = ['\t'.join(['chromosome', 'start', 'end', 'gc_content'])]
+
+if args.verbose: print('Load data')
+with open(args.input, "r") as f:
+    data = f.read().splitlines()
+
+if args.verbose: print('Calculate chromosome stats')
+chr_names, chr_borders = [], []
+
+for i in range(len(data)):
+    if '>' in data[i]:
+        chr_names.append(data[i].split('>')[1].split(' ')[0])
+        chr_borders.append(i)
+chr_borders = chr_borders + [len(data)]
+
+for i in range(len(chr_names)):
+
+    if not re.match("^(chr)?[0-9,X,Y]{1,2}$", chr_names[i]):
+        continue
+
+    if args.verbose: print('Calculating Chromosome {} - length = {} aa'.format(chr_names[i], 61*(chr_borders[i+1] - chr_borders[i])))
```

Comment:
Not sure what you do here. Why the length in aminoacids? It also seems you are estimate the chromosome length (in BPs) from the line-numbers. I say "estimate" because there is no guarantee that the lines contain the expected number of bases. Is "aa" a typo?

But this is just a diagnostic message. So maybe it's not so important.